src/util/AbstractAPI.js

```diff
@@ -64,7 +64,7 @@ Ext.define('Jarvus.util.AbstractAPI', {
             if (pageParams.apiHost && (urlMatch = pageParams.apiHost.match(/(^([a-zA-Z]+):\/\/)?([^/]+).*/))) {
                 me.setHost(urlMatch[3]);
                 me.setUseSSL('apiSSL' in pageParams ? Boolean(pageParams.apiSSL) : urlMatch[2] == 'https');
-            } else {
+            } else if (!me.getUseSSL()) {
```

Comment:
@themightychris Without this line the useSSL config wasn't taking effect since localhost or the OntrayPOS app doesn't have a https protocol to fall back on.